### ISSUE TITLE
Benchmarking against other differentiable convex optimization softwares

### ISSUE THREAD

**Issue by AKS1996** [AKS1996/DiffOpt.jl#11]
Although not now, some day down the road it might be good to benchmark our code (in terms of speed / supported problem types) against other existing solutions. Some possible candidates:
- https://github.com/cvxgrp/cvxpylayers
- https://github.com/locuslab/qpth

**Comment by AKS1996** [comment]
how could i forget
- https://github.com/cvxgrp/diffcp

**Comment by matbesancon** [comment]
yes agreed, setting up benchmark can be quite some work, a good starting point will be setting up tests (checking identical solutions) :)

**Comment by AKS1996** [comment]
using QPs from http://qplib.zib.de/
Open to any other references!
